lib/src/coap_media_type.dart

```diff
@@ -218,7 +219,7 @@ enum CoapMediaType {
   ),
 
   /// application/td+json
-  applicationTdJson(432, 'application', 'application/td+json'),
+  applicationTdJson(432, 'application', 'td+json'),
```

Comment:
Noticed that there was a bug in the code which caused this Content-Format's string value to become `application/application/td+json`. Added a fix in 890ad1a7abe061109f98b19aa6b52585eee4ce49 and included it in this PR.